JustSaying.UnitTests/AwsTools/MessageHandling/SqsNotificationListener/BaseQueuePollingTest.cs

```diff
@@ -44,7 +44,7 @@ public virtual async Task InitializeAsync()
 
             SystemUnderTest = CreateSystemUnderTest();
 
-            await When().ConfigureAwait(false);
+            await WhenAction().ConfigureAwait(false);
```

Comment:
Nit: Do we want this to be called `WhenActionAsync` to align it with `DisposeAsync` / `InitialiseAsync`?

Reply:
Ah ok, well `WhenAsync` would work unless anyone else has any strong feelings